src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs

```diff
@@ -43,9 +45,9 @@ private void RegisterObjectLimits()
       ModObjectLimit(UNIT_N05V_SHADOW_WEAVER_C_THUN_FACELESS_SHADOW_WEAVER, UNLIMITED);
       ModObjectLimit(UNIT_O001_TOL_VIR_STATUE_C_THUN_TOL_VIR_STATUE, 6);
 
-      ModObjectLimit(UNIT_H01K_SILITHID_OVERLORD_C_THUN_OVERLORD, 12);
       ModObjectLimit(UNIT_O02N_SILITHID_WASP_CTHUN, 24);
-      ModObjectLimit(UNIT_H01N_VILE_CORRUPTER_C_THUN_FACELESS_CORRUPTOR, 4);
+      ModObjectLimit(UNIT_H01N_VILE_CORRUPTER_C_THUN, 4);
+      ModObjectLimit(UNIT_SL2O_OBSIDIAN_ERADICATOR_CTHUN, 12);
 
       ModObjectLimit(FourCC("ushp"), UNLIMITED); //Undead Shipyard
```

Comment:
Let's get these using proper `Constants`. It's a new file so no point having tech debt straight away!

Reply:
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/47048283/97a7924b-5003-47f3-8648-298594ff009a)
These are all the boat stuff, since we were discussing cthun not having boats, i decided not to touch it for now!

Reply:
i can just remove it

Reply:
Makes sense. Nothing gets reviewed going from `dev` to `main`, so everything going into `dev` _either_ needs to be technically correct or have a ticket to finish it off.

So if this stuff is just a placeholder and you want to keep it in for now, you just need to create another Issue to clean it up (and as you say, maybe I end up picking up that Issue and doing some custom mechanic instead of the boats).